src/medals/main.cljs

```diff
@@ -0,0 +1,39 @@
+(ns medals.main
+  (:require
+   [medals.constants :refer [MEDALCOLORS SORTVALUES]]
+   [re-frame.core :as re-frame]
+   [medals.events :as events]
+   [medals.subs :as subs]
+   [medals.utils :refer [get-countries-with-total sort-countries]]))
+
+
+(defn tableHeaders [color selected-sort]
+  ^{:key color} [:th {:class (str "medal-colors" " "  (if (= selected-sort (:color_name color)) "selected" ""))}
+                 [:span {:style {:background-color (:hex_code color)} :on-click #(re-frame/dispatch [::events/change-selected-sort (:color_name color)])}]])
```

Comment:
Using inline styling is bad. Please use class instead.

and `:color_name` should be `:color-name`.